### ISSUE TITLE
Slides not on separate PDF pages; not using terminal dimensions

### ISSUE THREAD

**Issue by ghbrown** [mfontanini/presenterm-export#10]
Very similar to #4, but I figured I would start a new issue as there are some differences.

In addition to placing the page breaks in the middle of slides and the progress bar being mostly in the wrong spots, **the output seems to be entirely independent of the terminal size/shape and font size** (generally too large).

System info:
```
presenterm 0.8.0 (binary)
presenterm-export 0.2.4
tmux 3.4 (or 3.2)
gnome-terminal 3.40.3 (or kitty 0.35.2)
RHEL 9.4 (Plow)
```

I may not have access to this machine soon, so I wanted to file this while I can.

P.S. `presenterm` is sweet, I'm loving it.

**Comment by mfontanini** [comment]
Do you happen to be running tmux already and possibly have some tmux session open in another terminal as you run this? I saw the terminal size not being respected when this happens but didn't dig in much. From what I saw it looked like it could be a tmux bug since this does tell tmux what size to use, and it seems to work as long as you don't have tmux open in another terminal.

**Comment by ghbrown** [comment]
I had loads of tmux sessions open when I did my testing. Tomorrow I'll confirm that shutting them all down fixes this.

Could this be documented somewhere? I imagine this could be a relatively common gotcha.

**Comment by mfontanini** [comment]
Yes, I definitely should document this. I honestly saw this briefly while I was doing something else and completely forgot about it. If you manage to fix it by having a single time session open, please let me know!

**Comment by ghbrown** [comment]
Yep, that fixed it. Once I closed the open tmux sessions it picked up the dimensions and font size from the terminal calling the export command.

**Comment by mfontanini** [comment]
I created a ticket on the tmux repo and this should now be fixed in tmux 3.6a (or whatever comes after 3.5a). https://github.com/tmux/tmux/issues/4268
